packages/loader/container-loader/src/loader.ts

```diff
@@ -45,10 +45,6 @@ import { pkgVersion } from "./packageVersion";
 import { ProtocolHandlerBuilder } from "./protocol";
 import { DebugLogger } from "./debugLogger";
 
-function canUseCache(request: IRequest): boolean {
-	return request.headers?.[LoaderHeader.cache] === true;
```

Comment:
You should remove the `cache` LoaderHeader as well. It is still used in a bunch of places and those usages should be removed.

Reply:
It can't be removed yet for back-compat reasons because the old caching default was true
[AB#5047](https://dev.azure.com/fluidframework/internal/_workitems/edit/5047)

Reply:
I see. Is this just because of tests failing? The tests could pass in the underlying strin "fluid-cache" instead of using the types and the types could be removed.

Reply:
It's mainly for requesting the summarizer container
https://github.com/microsoft/FluidFramework/blob/main/packages/runtime/container-runtime/src/containerRuntime.ts#L3896

Reply:
Okay. You could change it to use the string "fluid-cache" directly. Maybe its fine to do it as a follow-up, but we would have to wait for the LTS version to be past current version. I will leave it to you.